providers/os/resources/smbios/win.go

```diff
@@ -14,7 +14,7 @@ import (
 const smbiosWindowsScript = `
 $bios = Get-WmiObject -class Win32_Bios
 $baseboard = Get-WmiObject Win32_BaseBoard
-$chassis = Get-WmiObject Win32_SystemEnclosure
+$chassis = @(Get-WmiObject Win32_SystemEnclosure)
```

Comment:
Note on PowerShell syntax:
- The `@()` syntax is used to (force) create an array in PowerShell.